### PR TITLE
[Catalog] Use light/dark snapshot version

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Description**

This PR fixes #
Use the version of snapshot based on theme 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -71,14 +71,14 @@ GEM
...
-      google-protobuf (~> 3.25)
+    sass-embedded (1.63.6-arm64-darwin)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.63.6-x64-mingw-ucrt)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.63.6-x86_64-darwin)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.63.6-x86_64-linux-gnu)
+      google-protobuf (~> 3.23)
     terminal-table (3.0.2)
```
Comment (aabidsofi19) [inline Gemfile.lock:82]: oops ..

**Comment by aabidsofi19** [comment]
@leecalcote

**Comment by leecalcote** [comment]
Ready to go?

**Comment by sudhanshutech** [comment]
@aabidsofi19 merge conflicts...
Please remove the changes in gemfile(it will override the ruby configs) and `.jekyll-metadata`

File: Gemfile.lock [hunk 2]
```diff
@@ -71,14 +71,14 @@ GEM
...
-    sass-embedded (1.69.6-x86_64-linux-gnu)
-      google-protobuf (~> 3.25)
+    sass-embedded (1.63.6-arm64-darwin)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.63.6-x64-mingw-ucrt)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.63.6-x86_64-darwin)
+      google-protobuf (~> 3.23)
+    sass-embedded (1.63.6-x86_64-linux-gnu)
+      google-protobuf (~> 3.23)
```
Comment (aabidsofi19) [inline Gemfile.lock:81]:
```suggestion
     sass-embedded (1.69.6-arm64-darwin)
      google-protobuf (~> 3.25)
    sass-embedded (1.69.6-x64-mingw-ucrt)
      google-protobuf (~> 3.25)
    sass-embedded (1.69.6-x86_64-darwin)
      google-protobuf (~> 3.25)
    sass-embedded (1.69.6-x86_64-linux-gnu)
      google-protobuf (~> 3.25)
```

**Comment by sudhanshutech** [comment]
oops, seems like screenshot are not visible in front page but it is inside the modal. Will you confirm @aabidsofi19 ?
image

**Review by MUzairS15 — APPROVED** [review]

**Comment by leecalcote** [comment]
Whee! @aabidsofi19 😁

**Pull request merged**